frontend/src/metabase/dashboard/actions/writeback.ts

```diff
@@ -44,9 +45,16 @@ export const closeActionParametersModal = createAction(
   CLOSE_ACTION_PARAMETERS_MODAL,
 );
 
+interface DashbardAttributes {
+  card_id?: number | null;
+  action?: WritebackAction | null;
+  parameter_mappings?: ActionParametersMapping[] | null;
+  visualization_settings?: ActionDashboardCard["visualization_settings"];
+}
```

Comment:
Small typo: it says `Dashbard` instead of `DashbOard`
P.S. dashbard should be a character class in our BI RPG game 😂

Reply:
can we refer to @tsmacdonald as the Dashbard henceforth?

Reply:
:chart_with_upwards_trend: :violin: